blu.js.org

CorruptBlu opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

CorruptBlu commented:
I've added a CNAME file:
https://i.imgur.com/vsZSKtX.png

MattIPv4 commented:
Please take a look at https://repl.it/talk/learn/How-to-use-a-custom-domain/8834, specifically the `Attaching to repl.it` to section. I believe we need a custom CNAME target once you add the custom domain to your repl.it project, not just thee project name that you currently have added to `cnames_active.js`.

CorruptBlu commented:
> Please take a look at https://repl.it/talk/learn/How-to-use-a-custom-domain/8834, specifically the `Attaching to repl.it` to section. I believe we need a custom CNAME target once you add the custom domain to your repl.it project, not just thee project name that you currently have added to `cnames_active.js`.

Alright, I changed it to the CNAME they gave me.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But it looks like your host is returning a 404?
This doesn't look like "reasonable content".

CorruptBlu commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But it looks like your host is returning a 404?
> This doesn't look like "reasonable content".

Huh, when I go to my site, https://blu.aakhilv.repl.co/, there isn't a 404 error. Repl.it says that it may take up to 48 hours for the CNAME record to set in.

CorruptBlu commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But it looks like your host is returning a 404?
> This doesn't look like "reasonable content".

Updated it. It works now. https://blu.js.org/

CorruptBlu commented:
Should I close this?

MattIPv4 commented:
Please leave it open so it can be merged. If closed, the requested subdomain will be removed.

indus commented:
I see. When I first visited it it just gave short note about something is starting up.Please leave this PR open - I will merge it soon.Am 25.08.2020 17:09 schrieb CorruptBlu <notifications@github.com>:

I've already added your requested subdomain to JS.ORGs zonefile. But it looks like your host is returning a 404?
This doesn't look like "reasonable content".

Huh, when I go to my site, https://blu.aakhilv.repl.co/, there isn't a 404 error. Repl.it says that it may take up to 48 hours for the CNAME record to set in.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

indus commented:
Welcome https://blu.js.org
